.github/workflows/push.yml

```diff
@@ -0,0 +1,23 @@
+name: "Push Checks"
+on:
+  push:
+    paths:
+      - "terraform/**"
```

Comment:
should also include this `push.yml` file

Reply:
I only want the workflow to run against changes in Terraform?

Reply:
It should also run when the pipeline changes, like adding a TF_VAR environment variable or changing the workspace

Reply:
All it does is Terraform FMT, on Terraform files. 🤔